src/App.js

```diff
@@ -1,25 +1,21 @@
 import React, { Component } from "react";
 import "./App.css";
-import Navbar from "./Components/NavBar/NavBar";
-import Footer from "./Components/Footer/Footer";
-import LandingPage from "./Components/LandingPage/LandingPage";
-import styled from "@emotion/styled";
+//import Footer from "./Components/Footer/Footer";
+import LandingPage from "./Components/Body/LandingPage";
+//import styled from "@emotion/styled";
 
-const PageContainer = styled.body`
-  width: 100%;
-  height: 100%;
-  background: #ffffff;
-`;
+// const PageContainer = styled.body`
```

Comment:
Also, anything commented out can be deleted. If need be we can always add it back. This will ensure our code does not become cluttered.